doc/source/user_guide/computation.rst

```diff
@@ -571,6 +571,29 @@ and we want to use an expanding window where ``use_expanding`` is ``True`` other
    3     3.0
    4    10.0
 
+.. versionadded:: 1.1
+
+For some problems knowledge of the future is available for analysis. For example, this occurs when
+each data point is a full time series read from an experiment, and the task is to extract underlying
+conditions. In these cases it can be useful to perform forward-looking rolling window computations.
+``FixedForwardWindowIndexer`` class is available for this purpose. This ``BaseIndexer`` subclass
```

Comment:
you can use the ref here, e.g. the one you are using in the whatsnew

Reply:
@jreback I've included a ref in whatsnew, pointing to this subsection of the user guide. If you meant something different, please tell me.

Reply:
no I mean a ref on FixedForwardIndexer here that points to the api (makes a clickable link), and one for BaseIndexer